pages/1.13/security/oss/login/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+navigationTitle: Login
+title: Login
+excerpt: Logging in to your DC/OS cluster
+menuWeight: 20
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# User login
+
+With DC/OS, login is the process of exchanging user credentials for a [DC/OS Authentication token](/1.13/security/oss/authentication/authentication-token/).
+
+Obtaining a DC/OS Authentication is required by any user before being able to use a DC/OS cluster. In DC/OS the life-time of an Authentication token is limited to five days. Once the Authentication token expires the user must log in again.
+
+DC/OS handles multiple user types. User accounts can be managed via the [IAM API](/1.13/security/oss/iam-api/); see [User Management](/1.13/security/oss/user-management/).
+
+Different login methods exist for different user types but each one yields a DC/OS authentication token:
+
+* **External user login**: External user accounts can only log in via single sign-on through Auth0 (using their Google, GitHub, or Microsoft credentials).
+* **Local user login**: A local user logs in by supplying a password which is compared to the password hash stored inside DC/OS.
+* **Service login**: A service logs in by supplying a short-lived "service login token" of which the signature is verified using the service account public key stored inside DC/OS.
+
+# User logout
+
+Users cannot be actively logged out of DC/OS. As long as an issued DC/OS Authentication tokens exists and is valid, the user that it was issued for can operate the DC/OS cluster. A user can however decide to delete any valid DC/OS Authentication token in their possession. The DC/OS CLI [auth logout](/1.13/cli/command-reference/dcos-auth/dcos-auth-logout/) command does exactly that.
+
+<p class="message--note"><strong>NOTE: </strong>There is no way to revoke access to a DC/OS cluster other than to wait until the authentication token expires.</p>
```

Comment:
Rewrite `There is no way to revoke access to a DC/OS cluster other than to wait until the authentication token expires.`

with `It is impossible to revoke access to a DC/OS cluster, but wait until the authentication token expires.`